styles/partial/home.css

```diff
@@ -66,49 +69,147 @@
 
 @media (max-width: 768px) {
   .home {
-    width: 100%;
+    width: auto;
   }
 
   .home__title {
-    font-size: 17px;
+    font-size: 24px;
     padding-bottom: 18px;
   }
   .home_button {
-    font-size: 12px;
+    font-size: 17px;
     padding: 5px 11px;
   }
   .home__content {
     padding-bottom: 25px;
-    line-height: 1.4;
-    font-size: 10px;
+    line-height: 1.7;
+    font-size: 17px;
+    text-align: center;
+    margin-right: auto;
+    margin-left: auto;
+  }
+  .home__image {
+    height: 530px;
+    width: 100%;
+  }
+
+  .home__wrapper {
+    text-align: center;
+    top: 37%;
+    display: block;
+    justify-content: center;
+  }
+
+  .home__image img {
+    max-width: 950px;
   }
 }
 @media (max-width: 460px) {
```

Comment:
Shall we create breakpoint values as variables? so that we can configure that in future.
Ex: mobile-max-width : 767.5 , tablet-min-width: 768
https://www.w3schools.com/css/css3_variables.asp